src/acore-wp-plugin/src/Components/ServerInfo/ServerInfoApi.php

```
@@ -15,12 +15,601 @@ public static function AccountCount() {
+    return 'acore_2fa_panel_unlock_' . $userId . '_' . hash('sha256', $token);
+}
+
+function acore_2fa_attempt_key(int $userId): string {
+    $token = function_exists('wp_get_session_token') ? (string) wp_get_session_token() : '';
+    return 'acore_2fa_verify_attempts_' . $userId . '_' . hash('sha256', $token);
+}
+
+/**
+ * Whether the current user has In-game 2FA active (account.totp_secret set).
+ */
+function acore_ingame_2fa_enabled(int $userId): bool {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$userId'. (undefined)

(UnusedFormalParameter)